xarray/core/dataset.py

```diff
@@ -802,6 +804,42 @@ def to_netcdf(self, path=None, mode='w', format=None, group=None,
     def __unicode__(self):
         return formatting.dataset_repr(self)
 
+    def info(self, buf=None):
+        """
+        Concise summary of a Dataset variables and attributes.
+        Parameters
+        ----------
+        buf : writable buffer, defaults to sys.stdout
+
+        See Also
+        --------
+        pandas.DataFrame.assign
+        netCDF's ncdump
+        """
+
+        if buf is None:  # pragma: no cover
+            buf = sys.stdout
+
+        lines = []
+        lines.append('xarray.Dataset {')
+        lines.append('dimensions:')
+        for name, size in self.dims.items():
+            lines.append('\t{name} = {size} ;'.format(name=name, size=size))
```

Comment:
I think these probably should all be unicode (using `u` literals), otherwise this will break for non-ASCII characters on Python 2. Take a look at what things look like in `formatting.py`.